### PR TITLE
Refactor config

### LINKED ISSUE #497: Refactor Config: Simple Renames
0. `analytics-mode: true`➔`disable-analytics: true` / [ref](https://github.com/ubiquity/ubiquibot/pull/383#discussion_r1254075401)
0. `chain-id: 1`➔`evm-network-id: 1` / [ref](https://github.com/ubiquity/ubiquibot/pull/383#discussion_r1254084562)
0. `max-concurrent-bounties: 2`➔`max-concurrent-assigns: 2` / [ref](https://github.com/ubiquity/ubiquibot/pull/383#discussion_r1254088651)
0. `incentive-mode: false`➔`comment-incentives: true` / [ref](https://github.com/ubiquity/ubiquibot/pull/383#discussion_r1254075947)

### PR DESCRIPTION
**Pull request by me505**
Resolves #497

This pr is made to refactor the `ubiquibot-config.yml` file

### CODE REVIEW

**Comment by me505** [comment]
@0xcodercrane can you also review this

File: src/configs/shared.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const DEFAULT_TIME_RANGE_FOR_MAX_ISSUE_ENABLED = true;
 export const DEFAULT_FOLLOWUP_TIME = "4 days"; // 4 days
 export const DEFAULT_DISQUALIFY_TIME = "7 days"; // 7 days
 
-export const DEFAULT_CHAIN_ID = 1; // ethereum
+export const DEFAULT_NETWORK_ID = 1; // ethereum
```
Comment (pavlovcik) [inline src/configs/shared.ts:27]:
```suggestion
export const DEFAULT_EVM_NETWORK_ID = 1; // ethereum
```

Might be more clear what type of network we're talking about!

File: src/utils/private.ts [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const getWideConfig = async (context: Context) => {
   const privateKeyDecrypted = parsedOrg && parsedOrg[KEY_NAME] ? await getPrivateKey(parsedOrg[KEY_NAME]) : undefined;
 
   const configData = {
-    chainId: getChainId(parsedRepo, parsedOrg),
+    networkId: getNetworkId(parsedRepo, parsedOrg),
```
Comment (pavlovcik) [inline src/utils/private.ts:129]:
```suggestion
    evmNetworkId: getEvmNetworkId(parsedRepo, parsedOrg),
```

File: .github/ubiquibot-config.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-chain-id: 100
+evm-network-id: 1
```
Comment (pavlovcik) [inline .github/ubiquibot-config.yml:2]:
```suggestion
evm-network-id: 100
```

Gnosis Chain!

**Comment by me505** [comment]
@0xcodercrane have you got any updates for any of my prs?

**Comment by me505** [comment]
@pavlovcik can you resolve this conflicts
Thank you

**Comment by 0xcodercrane** [comment]
> @pavlovcik can you resolve this conflicts Thank you

Do you have any blockers to resolve conflicts? Bounty admins can definitely suggest code changes that might generate a couple of conflicts but if its not an impossible one and doesn't require lots of work, hunters should be responsible for resolving conflicts.

**Comment by me505** [comment]
@0xcodercrane sorry about that.
ill try to be less dependent

**Review by 0xcodercrane — DISMISSED** [review]

**Comment by 0xcodercrane** [comment]
The codebase looks good. The PR requires changes in configuration file so I am tagging @rndquu for a review and collaboration in configuration changes in ubiquity/ubiquibot, ubiquibot/staging, ubiquibot/production repos. I guess everything should happen at a same time.

**Comment by me505** [comment]
thanks

**Comment by rndquu** [comment]
@me505 The original issue describes renaming `analytics-mode` to `disable-analytics`. Is it implemented?

**Comment by me505** [comment]
sorry about that!
i thought because of [this](https://github.com/ubiquity/ubiquibot/pull/383#discussion_r1259810255) that i shouldnt implement it

**Comment by rndquu** [comment]
> The codebase looks good. The PR requires changes in configuration file so I am tagging @rndquu for a review and collaboration in configuration changes in ubiquity/ubiquibot, ubiquibot/staging, ubiquibot/production repos. I guess everything should happen at a same time.

There is a separate [issue](https://github.com/ubiquity/ubiquibot/issues/357) exactly for this "renaming" case

Initially the current issue should've been dependent on this [one](https://github.com/ubiquity/ubiquibot/issues/357). This way we could perform a smooth update.

Anyway, when the current PR is merged and a new bot's production version is released the following PRs should be merged:
- https://github.com/ubiquity/ubiquibot-config/pull/3
- https://github.com/ubiquibot/ubiquibot-config/pull/2

**Review by rndquu — CHANGES_REQUESTED** [review]
If `disable-analytics` is `true` then [here](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/wildcard/analytics.ts#L48) `!analyticsMode` is `false` thus actually enabling analytics instead of disabling it

**Review by rndquu — CHANGES_REQUESTED** [review]
If partner has `disable-analytics: true` then [here](https://github.com/ubiquity/ubiquibot/blob/development/src/handlers/wildcard/analytics.ts#L48) `!analyticsMode` is `false` thus enabling analytics instead of disabling it

I guess we need to refactor `analyticsMode` to `disableAnalytics` in the bot's code

File: .github/ubiquibot-config.yml [hunk 4]
```diff
@@ -29,7 +29,7 @@ priority-labels:
   - name: "Priority: 4 (Emergency)"
     weight: 5
 auto-pay-mode: true
-analytics-mode: true
+disable-analytics: true
```
Comment (pavlovcik) [inline .github/ubiquibot-config.yml]:
```suggestion
```

This should be enabled by default, right?

Comment (me505) [reply .github/ubiquibot-config.yml]: [ref](https://github.com/ubiquity/ubiquibot/pull/512#pullrequestreview-1546975867)

Comment (pavlovcik) [reply .github/ubiquibot-config.yml]: Okay if the value is `undefined` then the bot should enable analytics. 

I'm pretty sure in the specification I wrote that the value should be hidden from the default configuration. 

However analytics should always be on UNLESS we deliberately add the property to disable it.

Comment (me505) [reply .github/ubiquibot-config.yml]: ohhh

**Comment by me505** [comment]
ill assume you want analytics on

**Review by pavlovcik — APPROVED** [review]
My only hesitation is if the logic is implemented correctly: 

- If `disable-analytics` property is not on the config, then analytics should be enabled.

- If `disable-analytics: true` then analytics should be disabled
- If `disable-analytics: false` then analytics should be enabled.

@rndquu @0xcodercrane what do you guys think?

**Comment by rndquu** [comment]
> My only hesitation is if the logic is implemented correctly:
> 
> * If `disable-analytics` property is not on the config, then analytics should be enabled.
> * If `disable-analytics: true` then analytics should be disabled
> * If `disable-analytics: false` then analytics should be enabled.
> 
> @rndquu @0xcodercrane what do you guys think of the code?

PR looks good, the logic is implemented correctly

**Review by rndquu — APPROVED** [review]

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**